types/chrome/index.d.ts

```diff
@@ -4790,30 +4790,35 @@ declare namespace chrome.identity {
      *  * Removes all OAuth2 access tokens from the token cache
      *  * Removes user's account preferences
      *  * De-authorizes the user from all auth flows
+     * If `callback` is not provided, the function returns a Promise when the state has been cleared.
      * @since Chrome 87.
      * @param callback Called when the state has been cleared.
      * The parameter should be a function that looks like this:
      * () => {...};
```

Comment:
These lines are redundant and I'll probably send a PR in the future to remove it. TSDoc should not repeat types